packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -23,14 +24,15 @@ export default class RatingBar extends PureComponent {
     score: 0,
     maxScore: 5,
     RatingItem: Star,
+    color: 'royalblue',
```

Comment:
Do we really need this default?

Reply:
Not really, but since i made it a required prop, tests stared throwing warnings.  This was the easy way out.  I'll update the tests instead if you think it's important.

Reply:
I just didn't want to force a color. Maybe if anything we should just default to `black` or or something.